packages/expo/src/__tests__/Haptic-test.ts

```diff
@@ -0,0 +1,102 @@
+import { UnavailabilityError } from 'expo-errors';
+import { Platform } from 'expo-core';
+import * as Haptic from '../Haptic/Haptic';
+
+const unavailableMessage = `is unavailable on ${Platform.OS}`;
+
+async function executeFailingMethod(method, name) {
+  try {
+    await method();
+    expect(name).toBe('a failing method');
+  } catch (error) {
+    expect(error instanceof UnavailabilityError).toBeTruthy();
+  }
+}
+
+describe('Haptic', () => {
+  if (Platform.OS === 'ios') {
+    describe('selection()', () => {
+      it("Doesn't throw an error", async () => {
+        let resultingError;
+        try {
+          await Haptic.selection();
+        } catch (error) {
+          resultingError = error;
+        }
+        expect(resultingError).toBeUndefined();
```

Comment:
If you want to test whether `await Haptic.selection()` doesn't fail, there is simpler solution 🙂 
```javascript
it("Doesn't throw an error", async () => {
  await Haptic.selection();
});
```
There is actually no need to use `expect`s here.